.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -158,6 +158,7 @@ Release Week Checklist:
 - [ ] **Check for presence of jdk8u aarch32 GA tag and mirror it** [Upstream Git repo](https://github.com/openjdk/aarch32-port-jdk8u) - [Mirror job](https://ci.adoptium.net/view/git-mirrors/job/git-mirrors/job/adoptium/job/git-skara-aarch32-jdk8u/)
 - [ ] **Do all of the above for the jdk8u/aarch32 build: Ensure to specify overridePublishName param**
 - [ ] **Archive/upload all TCK results**
+- [ ] **Archive/upload all AQA results** [Publish AQA test results](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md)
```

Comment:
Too bad we can not hyperlink directly to an anchor at that page more specific than https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#after-build-pipeline-finished as someone then has to search for the info.

Reply:
Yes, I was thinking to refer by code https://github.com/adoptium/temurin-build/blob/master/RELEASING.md?plain=1#L304, but feels not good either.